test/Dialect/TorchConversion/torch-backend-to-linalg-on-tensors-no-mlprogram.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: torch-mlir-opt -p 'builtin.module(torch-backend-to-linalg-on-tensors-backend-pipeline{use-mlprogram=0})' -split-input-file %s | FileCheck %s
+
+// CHECK-NOT: ml_program.global private mutable @global_seed(dense<0> : tensor<i64>) : tensor<i64>
```

Comment:
Nit: Is it possible to only check that there are no references to the `ml_program` dialect? This kind of test can be very easy to break.